apps/evm/src/components/Tooltip/index.tsx

```diff
@@ -1,22 +1,87 @@
-import { Global } from '@emotion/react';
-import MuiTooltip, { type TooltipProps as MUITooltipProps } from '@mui/material/Tooltip';
+import {
+  Arrow,
+  Content,
+  Portal,
+  Provider,
+  type TooltipContentProps,
+  Tooltip as TooltipPrimitive,
+  type TooltipProps as TooltipPrimitiveProps,
+  Trigger,
+} from '@radix-ui/react-tooltip';
+import { Modal } from 'components/Modal';
+import { useBreakpointUp } from 'hooks/responsive';
+import React, { useState } from 'react';
+import { cn } from 'utilities';
 
-import { useStyles } from './styles';
-
-export interface TooltipProps extends Omit<MUITooltipProps, 'children'> {
-  title: string | React.ReactElement;
-  children: React.ReactNode;
+export interface TooltipProps extends TooltipPrimitiveProps {
+  className?: string;
+  content: string | React.ReactElement;
+  side?: TooltipContentProps['side'];
 }
 
-export const Tooltip = ({ children, placement = 'top', ...rest }: TooltipProps) => {
-  const styles = useStyles();
+const TooltipContent = React.forwardRef<
+  React.ElementRef<typeof Content>,
+  React.ComponentPropsWithoutRef<typeof Content>
+>(({ className, sideOffset = 4, ...props }, ref) => (
+  <Portal>
+    <Content
+      ref={ref}
+      onClick={e => e.stopPropagation()}
+      sideOffset={sideOffset}
+      className={cn(
+        'flex z-50 overflow-hidden rounded-xl bg-lightGrey p-2 max-w-75 text-sm text-primary-foreground animate-in fade-in-0 zoom-in-95',
+        'data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=closed]:zoom-out-95 data-[side=bottom]:slide-in-from-top-2',
+        'data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 data-[side=top]:slide-in-from-bottom-2',
+        className,
+      )}
+      {...props}
+    />
+  </Portal>
+));
+TooltipContent.displayName = Content.displayName;
+
+export const Tooltip = ({
+  className,
+  content,
+  children,
+  delayDuration = 200,
+  side,
+  ...props
+}: TooltipProps) => {
+  const [isTooltipOpened, setIsTooltipOpened] = useState(false);
+  const isMdOrUp = useBreakpointUp('md');
+
+  const handleToggleDropdown = () => setIsTooltipOpened(!isTooltipOpened);
 
   return (
-    <>
-      <Global styles={styles} />
-      <MuiTooltip arrow placement={placement} enterTouchDelay={0} {...rest}>
-        <span>{children}</span>
-      </MuiTooltip>
-    </>
+    <Provider>
+      <TooltipPrimitive delayDuration={delayDuration} {...props}>
+        <Trigger
+          asChild
+          className={className}
+          onClick={e => {
+            if (!isMdOrUp) {
+              setIsTooltipOpened(true);
+              e.preventDefault();
+            }
+          }}
+        >
+          <div>{children}</div>
+        </Trigger>
+        <TooltipContent side={side} className={cn('block p-3', !isMdOrUp && 'hidden')}>
+          {content}
+          <Arrow className="fill-lightGrey" width={14} height={7} />
```

Comment:
nit: could you define the width and height using classes instead?
```suggestion
          <Arrow className="fill-lightGrey w-[14px] h-[7px]" />
```